### PR TITLE
Feature/bulma 0 7 modifiers support

### PR DESCRIPTION
**Pull request by MangelMaxime**
@selketjah If you could review the section "Modifiers" of the doc it would be awesome :)

### CODE REVIEW

File: docs/src/Fulma/Modifiers/Basics.fs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+let view =
+    Render.docPage [
+        Render.docSection
+            """
+# Modifiers - Basics
+
+*[Bulma documentation](https://bulma.io/documentation/modifiers/)*
+
+You can add `Modifiers` to any Fulma elements or components.
```
Comment (selketjah) [inline docs/src/Fulma/Modifiers/Basics.fs]: should be single: any Fulma element or component

File: docs/src/Fulma/Modifiers/Colors.fs [hunk 2]
```diff
@@ -65,7 +65,7 @@ let view =
     Render.docPage [
         Render.contentFromMarkdown
             """
-# Modifiers
+# Modifiers - Colors & Shades
 
 ## Colors
```
Comment (selketjah) [inline docs/src/Fulma/Modifiers/Colors.fs]: color & shade isn't technically a modifier you can add.
you add the modifiers BackgroundColor and TextColor, which can be a color or a shade.
and you can add them to components also if I am not mistaken.

File: docs/src/Fulma/Modifiers/Typography.fs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            [ Text.span [ Modifiers [ Modifier.TextWeight TextWeight.Bold ] ]
+                [ str "Bold" ] ] ]
+
+let view =
+    Render.docPage [
+        Render.contentFromMarkdown
+            """
+# Modifiers - Typography
+
+Change the **size** and **color** of the text for one or multiple viewport width.
```
Comment (selketjah) [inline docs/src/Fulma/Modifiers/Typography.fs]: color should be alignment?

Comment (selketjah) [inline docs/src/Fulma/Modifiers/Typography.fs]: maybe add an example of changing the size on multiple viewports too

File: docs/src/Fulma/Modifiers/Colors.fs (hunk 2 above)
Comment (MangelMaxime) [reply docs/src/Fulma/Modifiers/Colors.fs]: Yes but I don't have a good small name :)

I kind of mimic Bulma docs here. They speak about `Color helpers`

File: docs/src/Fulma/Modifiers/Typography.fs (hunk 3 above)
Comment (MangelMaxime) [reply docs/src/Fulma/Modifiers/Typography.fs]: Can't you let me be lazy ? 😠 

:trollface: :trollface: :trollface: 

Ok ok, I will add the missing section ^^

**Comment by MangelMaxime** [comment]
@selketjah Thank you for the review. I am fixing the different part and will publish the new version directly :)

File: docs/src/Fulma/Modifiers/Typography.fs (hunk 3 above)
Comment (MangelMaxime) [reply docs/src/Fulma/Modifiers/Typography.fs]: Ok it's possible that thanks to the sample we avoid a bug in the next release :D 🤷‍♂️

Comment (selketjah) [reply docs/src/Fulma/Modifiers/Typography.fs]: you can thank me appropriately later :-p

**Pull request merged**